python/setup.py

```diff
@@ -86,7 +86,7 @@ def get_llvm_package_info():
         return Package("llvm", "LLVM-C.lib", "", "LLVM_INCLUDE_DIRS", "LLVM_LIBRARY_DIR", "LLVM_SYSPATH")
     # use_assert_enabled_llvm = check_env_flag("TRITON_USE_ASSERT_ENABLED_LLVM", "False")
     # release_suffix = "assert" if use_assert_enabled_llvm else "release"
-    rev = "b1115f8c"
+    rev = "49af6502"
```

Comment:
Can we read the value in llvm-hash.txt?  (And update the README.md that currently points here?)